lib/Chat/ChatManager.php

```diff
@@ -730,26 +725,14 @@ public function getMessagesById(Room $chat, array $commentIds): array {
 		return $comments;
 	}
 
-	public function queueRemindLaterBackgroundJob(Room $chat, IComment $comment, Attendee $attendee, int $timestamp): void {
-		$this->jobList->add(ChatMessageReminder::class, [
-			'execute-after' => $timestamp,
-			'token' => $chat->getToken(),
-			'message_id' => $comment->getId(),
-			'message_actor_type' => $comment->getActorType(),
-			'message_actor_id' => $comment->getActorId(),
-			'user' => $attendee->getActorId(),
-		]);
-	}
-
-	public function dismissReminderNotification(Room $chat, IComment $comment, Attendee $attendee): void {
-		$notification = $this->notificationManager->createNotification();
-		$notification->setApp(Application::APP_ID)
-			->setUser($attendee->getActorId())
-			->setObject('reminder', $chat->getToken())
-			->setMessage('reminder', [
-				'commentId' => $comment->getId(),
-			]);
-		$this->notificationManager->markProcessed($notification);
+	/**
+	 * Get messages by ID
+	 *
+	 * @param int[] $commentIds
+	 * @return IComment[] Key is the message id
+	 */
+	public function getMessagesById(array $commentIds): array {
+		return $this->commentsManager->getCommentsById(array_map('strval', $commentIds));
```

Comment:
I wonder if conversion to `string` shouldn't be done at lower level.
It seems we always use `int` everywhere else.

Probably also out of scope of this pull request

Reply:
commentsManager is from server and it operates based on strings (for legacy reasons, I can tell you the story another day). So when interacting with that we cast to string and towards talks chatManager we always use ints because that is how the DB works.